[ERROR] /home/travis/build/nutzam/nutzboot/nutzboot-core/src/main/java/org/nutz/boot/config/impl/PropertiesConfigureLoader.java:[27,18] cannot find symbol
[ERROR]   symbol:   class UnsupportedEncodingException
[ERROR]   location: class org.nutz.boot.config.impl.PropertiesConfigureLoader

--- a/nutzboot-core/src/main/java/org/nutz/boot/config/impl/PropertiesConfigureLoader.java
+++ b/nutzboot-core/src/main/java/org/nutz/boot/config/impl/PropertiesConfigureLoader.java
@@ -4,6 +4,7 @@ import java.io.File;
 import java.io.FileInputStream;
 import java.io.InputStream;
 import java.util.ArrayList;
+import java.io.UnsupportedEncodingException;
 
 import org.nutz.ioc.impl.PropertiesProxy;
 import org.nutz.lang.Streams;

